src/NLog.Layouts.GelfLayout/GelfConverter.cs

```diff
@@ -55,15 +51,30 @@ public JObject GetGelfJson(LogEventInfo logEventInfo, string facility)
             //Convert to JSON
             var jsonObject = JObject.FromObject(gelfMessage);
 
-            //Add any other interesting data to LogEventInfo properties
-            logEventInfo.Properties.Add("LoggerName", logEventInfo.LoggerName);
-
             //We will persist them "Additional Fields" according to Gelf spec
-            foreach (var property in logEventInfo.Properties)
+            if (logEventInfo.Properties.Count > 0)
             {
-                AddAdditionalField(jsonObject, property);
+                foreach (var property in logEventInfo.Properties)
```

Comment:
No that would be a breaking change. Because before it was adding directly to the LogEventInfo.Properties causing exception-details and loggername to be added as last entries. By not modifying the LogEventInfo.Properties, then I need to add them to end manually.

Reply:
ok